### PR TITLE
[Feature] Support MViT and add checkpoints.

### PR DESCRIPTION
**Pull request by mzr1996**
## Motivation

Support [Multi-scale ViT v2](https://github.com/facebookresearch/mvit)

## Modification

As the title.

### CODE REVIEW

File: mmcls/models/backbones/mvit.py [hunk 1]
```diff
@@ -0,0 +1,700 @@
...
+    if norm is not None:
+        x = norm(x)
+
+    if ndim == 3:
+        x = x.squeeze(1)
+
+    return x, out_size
+
+
+class MultiScaleAttention(BaseModule):
```
Comment (tonysy) [inline mmcls/models/backbones/mvit.py:166]: Maybe this class can be moved to utils/attention.py

Comment (mzr1996) [reply mmcls/models/backbones/mvit.py:166]: I think we can move it when another backbone requires this structure.

File: configs/mvit/README.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+| MViTv2-small\* | From scratch |   34.87   |   7.00   |   83.63   |   96.51   | [config](https://github.com/open-mmlab/mmclassification/blob/master/configs/mvit/mvitv2-small_8xb256_in1k.py) | [model](https://download.openmmlab.com/mmclassification/v0/mvit/mvitv2-small_3rdparty_in1k_20220722-986bd741.pth) |
+| MViTv2-base\*  | From scratch |   51.47   |  10.20   |   84.34   |   96.86   | [config](https://github.com/open-mmlab/mmclassification/blob/master/configs/mvit/mvitv2-base_8xb256_in1k.py) | [model](https://download.openmmlab.com/mmclassification/v0/mvit/mvitv2-base_3rdparty_in1k_20220722-9c4f0a17.pth) |
+| MViTv2-large\* | From scratch |  217.99   |  42.10   |   85.25   |   97.14   | [config](https://github.com/open-mmlab/mmclassification/blob/master/configs/mvit/mvitv2-large_8xb256_in1k.py) | [model](https://download.openmmlab.com/mmclassification/v0/mvit/mvitv2-large_3rdparty_in1k_20220722-2b57b983.pth) |
+
```
[the remark this answers is not in the document]
Comment (mzr1996) [reply configs/mvit/README.md:32]: Good idea, but the `README.md` is standard for the parsing of the open-platform. If we want to add more parts, maybe design a new standard and add them in another PR.

File: mmcls/models/backbones/mvit.py (hunk 1 above)
Comment (tonysy) [reply mmcls/models/backbones/mvit.py:166]: OK

File: configs/mvit/README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (tonysy) [reply configs/mvit/README.md:32]: OK

**Review by tonysy — APPROVED** [review]
LGTM

**Pull request merged**